modules/web/src/app/shared/components/chip-list/component.ts

```diff
@@ -112,7 +112,14 @@ export class ChipListComponent implements OnChanges, OnDestroy, ControlValueAcce
     }
 
     event.chipInput?.clear();
-    this.tags.push(value);
+    const delimiters = [',', ' '];
+    const splitRegex = new RegExp(`[${delimiters.join('')}]`);
+    if (splitRegex.test(value)) {
+      const multiTags = value.split(splitRegex);
+      multiTags.forEach(val => this.tags.push(val.trim()));
```

Comment:
If both `,` and ` ` appear together in the string then it `split` will give an extra empty value . For example, `123, 456` would return `['123', '', '456']`. So we should add a filter on splitted values:
```suggestion
      const multiTags = value.split(splitRegex);
      multiTags.filter(val => !!val.trim()).forEach(val => this.tags.push(val.trim()));
```